src/server/v2.0/handler/gc.go

```diff
@@ -197,7 +197,19 @@ func (g *gcAPI) GetGC(ctx context.Context, params operation.GetGCParams) middlew
 }
 
 func (g *gcAPI) GetGCLog(ctx context.Context, params operation.GetGCLogParams) middleware.Responder {
-	log, err := g.gcCtr.GetTaskLog(ctx, params.GcID)
+	query, err := g.BuildQuery(ctx, nil, nil, nil)
+	if err != nil {
+		return g.SendError(ctx, err)
+	}
+	query.Keywords["ExecutionID"] = params.GcID
+	tasks, err := g.gcCtr.ListTasks(ctx, query)
+	if err != nil {
+		return g.SendError(ctx, err)
+	}
+	if len(tasks) == 0 {
+		return g.SendError(ctx, errors.New(nil).WithCode(errors.NotFoundCode).WithMessage("garbage collection %d log is not found", params.GcID))
+	}
+	log, err := g.gcCtr.GetTaskLog(ctx, tasks[0].ID)
```

Comment:
How about removing GetTaskLog and ListTasks and implement `GetGCLog(ctx context.Context, executionID int64)` in the gc controller ?

Reply:
as discussed with @ywk253100 , we should not provide specific method in the GC handler, so that it can be easy to migrate the new GC api in furture.

Reply:
But, I think GetTaskLog, is the method of the task controller/manager.